communication/slack_channels.md

```diff
@@ -0,0 +1,62 @@
+# Slack Channels
+
+Please find a list here of Slack channels at source{d}.
+
+Slack channels are encouraged to be open for everyone to join. Private channels for professional reasons are highly discouraged (have fun with private personal slack channels).
+
+### General channels
+
+* `#general`
+* `#offtopic`
+* `#officechat`
+  * Physical office related discussions
+* `#offtopic-`
+  * Search in slack for `#offtopic-` and you'll find different offtopic channels like `#offtopic-music`
+* `#osd`
+  * `#osd` stands for Open Source Days, people share what they'll be working on during their OSD and share the results/demos
+* `#twitter`
+  * Source{d} mentions and tweets
+
```

Comment:
We have channels such as #brewing (source{d} made beer and the alike), #python (for python-focused discussions, links and Q&A) and #spanish (for Spanish-focused discussions, classes and Q&A). Should these be listed as well, become `offtopic-xxx`…? Or not really necessary to list everything here?

Reply:
I would make them #oddt

Reply:
massive renaming does not make a lot of sense.
Mentioning some prominent ones might be worth, like #brewing with a link to https://github.com/src-d/homebrew etc